tasmota/tasmota_configurations_ESP32.h

```diff
@@ -52,6 +52,9 @@
 #define FALLBACK_MODULE        ODROID_GO         // [Module2] Select default module on fast reboot where USER_MODULE is user template
 
 #define USE_ODROID_GO                            // Add support for Odroid Go
+#define USE_UFILESYS
+#define USE_SDCARD
+#define SDCARD_CS_PIN     22
```

Comment:
NO GO!!!!!

Learn to use the embedded template and set the corrrect gpio there. These defines are a gemu nightmare resulting in unmanageable code.

Reply:
I just did a commit where this SD card has been set for the odroid

Reply:
Thx, i have to get use to change modules (again) :-)